test/jqLiteSpec.js

```diff
@@ -635,6 +635,56 @@ describe('jqLite', function() {
       expect(select.attr('multiple')).toBe('multiple');
     });
 
+    it('should set respective properties when getting/setting boolean attributes', function() {
```

Comment:
As far as I understand, it is actually the browser that does that, not jqLite/`.attr()`. 
We should either remove the test (if we don't consider it to be jqLite's job, or make sure jqLite always sets the properties (instead of relying on the browsers to do it).